### ISSUE TITLE
crash + softlock in san francisco boxing ring

### ISSUE THREAD

**Issue by AXZIOM11** [BGforgeNet/Fallout2_Restoration_Project#240]
### What happened
While entering the last of the six martial artists fights the game either crashes, or ziggy disappears, i don't think there is a way to fix it other than go back to an earlier save so thats what i will do
### Video
https://github.com/BGforgeNet/Fallout2_Restoration_Project/assets/100983356/b9a141fb-8c46-4d95-8668-bb5831dbfde3
Note: usually the crash happens before loading into the next fight, for some reason when i recorded this it was after
### Savegame
[SLOT129.zip](https://github.com/BGforgeNet/Fallout2_Restoration_Project/files/12732325/SLOT129.zip)

**Comment by burner1024** [comment]
I can't get the crash, but I do always get the softlock.
Did starting from earlier save fix it?

**Comment by AXZIOM11** [comment]
yes, i had to go pretty far back though because i didn't have any recent saves

**Comment by burner1024** [comment]
It might be worth it to add rotating autosaves on map enter/leave, like some other games do. Either natively with sfall, or expose save game functionality to scripts, and have a global script do it.
Maybe float this idea in sfall repo.
